File: 은지/LV2/JadenCase 문자열 만들기.js
function solution(s) {
  var answer = '';
  for (let i=0; i<s.length; i++) {
    if (i === 0) {
      answer += s[i].toUpperCase()
    }
    else if (s[i-1] === " ") {
      answer += s[i].toUpperCase()
    }
    else {
      answer += s[i].toLowerCase()
    }
  }
  return answer;
}

File: 은지/LV2/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''
    for i in range(len(s)):
        if i == 0:
            answer += s[i].upper()
        elif s[i-1] == " ":
            answer += s[i].upper()
        else:
            answer += s[i].lower()
    return answer

s = "3people unFollowed me"
print(solution(s))